modules/monitoring-accessing-the-metrics-of-your-service-as-a-developer.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="accessing-the-metrics-of-your-service-as-a-developer_{context}"]
+= Accessing the metrics of your service as a developer
+
+Once you have enabled monitoring your own services, deployed a service, and set up metrics collection for it, you can access the metrics of the service as a developer or as a user with view permissions for the project.
```

Comment:
s/Once you have enabled monitoring your own services/_After_ you have enabled monitoring _of_ your own services/

**once** _adv_
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

Reply:
"set up metrics collection for it" - what do you mean by "it"? "service"? Whatever it is, I would spell it out.

Reply:
I tried to avoid too many "service"s in a row. :/ Still, applied your suggestion.